contentcuration/contentcuration/frontend/shared/vuetify/theme.js

```diff
@@ -12,6 +12,7 @@ export default function theme() {
     greenSuccess: '#4db6ac',
     topic: colors.grey.base,
     video: '#283593',
+    image: '#283593', // IDK what this is supposed to be
```

Comment:
Ah yes, this is for the content kind icon background colors
![image](https://user-images.githubusercontent.com/7447496/75597630-68788f80-5a4b-11ea-9973-84f880ab975f.png)

Reply:
What is it supposed to be for the image type? It's not in the graphic above.

Reply:
It was supposed to be for the slideshow, so it's probably okay to remove it